ml-agents-envs/mlagents/envs/env_manager.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict, NamedTuple, Optional
+from mlagents.envs import AllBrainInfo, BrainParameters, Policy, ActionInfo
+
+
+class StepInfo(NamedTuple):
+    last_all_brain_info: Optional[AllBrainInfo]
+    current_all_brain_info: AllBrainInfo
+    all_action_infos: Optional[Dict[str, ActionInfo]]
```

Comment:
Can you add a comment here on what the key is? brain name? Alternatively, naming it something like `action_info_by_brain_name` or `brain_name_to_action_info` would make it clearer.

Reply:
👍 -- I think changing the name is the clearest thing